api/spec/controllers/spree/api/checkouts_controller_spec.rb

```diff
@@ -308,7 +308,9 @@ module Spree
       context "#{action}" do
         context "with order in confirm state" do
           subject do
-            api_put action, params
+            ActiveSupport::Deprecation.silence do
```

Comment:
No, that shouldn't be too hard to do. I can just toss in a conditional. It'll all get deleted once we remove the deprecation action anyway.